cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -397,10 +370,7 @@ bool cros_gralloc_driver::is_video_format(const struct cros_gralloc_buffer_descr
 
 bool cros_gralloc_driver::use_ivshm_drv(const struct cros_gralloc_buffer_descriptor *descriptor)
 {
-	if ((descriptor->use_flags & BO_USE_SCANOUT) &&
-            (descriptor->width == IVSH_WIDTH) && (descriptor->height == IVSH_HEIGHT))
-		return true;
-        return false;
+	return (descriptor->width == IVSH_WIDTH) && (descriptor->height == IVSH_HEIGHT);
```

Comment:
Yue changed the code that only scanout buf need to be allocated by ivsh node. Is this logical changed?

Reply:
Hi @yhe39, do we need SCANOUT flag to use ivsh node?

Reply:
> Hi @yhe39, do we need SCANOUT flag to use ivsh node?

Yes, by design, only the scanout buffer with special resolution use ivshm device to allocate, others need use intel gpu to allocate.